src/modules/core/components/Fields/Input/InputComponent.tsx

```diff
@@ -97,7 +97,7 @@ const InputComponent = ({
       onChange={onChange}
       placeholder={placeholder}
       ref={innerRef}
-      defaultValue={value}
+      value={value || ''}
```

Comment:
Why was this changed? 

Note that using `value` as opposed to `defaultValue` will make the input go in and out of React controlled / uncontrolled state.

Also, why does this default to an empty string? This is why we have `initialValues`. Please remove this.

Reply:
it wont go in and out of React controlled/uncontrolled, because you have default value passed. This is the only way we can fill out form with current domain data. Unless you know about better solution to fix this problem

Reply:
>  This is the only way we can fill out form with current domain data. 

But then how did it work for the colony edit modal then? When I implemented that, it worked if I just passed in `defaultValue={value}`